config/default.schema.json

```diff
@@ -1306,6 +1306,32 @@
 			"default": false,
 			"description": "Enables feature that allows the insecure LDAP URL (with ldap:// protocol)."
 		},
+		"ADMIN_API": {
+			"type": "object",
+			"description": "Configuration of the schulcloud-server's admin API.",
+			"properties": {
+				"ENABLED": {
+					"type": "boolean",
+					"default": true,
+					"description": "Flag to turn on/off the Admin API."
+				},
+				"PORT": {
+					"type": "number",
+					"default": 4030,
+					"description": "Port of the exposed Admin API server."
+				},
+				"ALLOWED_API_KEYS": {
+					"type": "string",
+					"default": "",
+					"description": "Allowed Admin API keys (for accessing the Admin API)."
+				}
+			},
+			"default": {
+				"ENABLED": true,
+				"PORT": 4030,
```

Comment:
Can be removed i think.

Reply:
Well this was added as duplicate because there was some problem with running app. I can delete it now and see what will happen

Reply:
done

Reply:
e2e failing don't know why it doesn't read values properly without duplication

Reply:
ok it looks like it works now